.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,21 @@
+---
+name: Bug report
+about: Tell us about a problem you are experiencing
+labels: carvel-triage
```

Comment:
We should also add `bug` when it is opened.

Reply:
done